packages/resolve-scripts/configs/schema.resolve.config.json

```diff
@@ -0,0 +1,132 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "adapter": {
+      "type": "object",
+      "properties": {
+        "adapter": {
+          "type": "string"
+        },
+        "options": {
+          "type": "object"
+        }
+      },
+      "additionalProperties": false
+    },
+    "jwtCookie": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "maxAge": {
+          "type": "integer"
+        }
+      },
+      "additionalProperties": false
```

Comment:
Field `jwtCookie` validation is meant to have only two fields: `name` and `maxAge`, correct? But `applyJwtCookie` meant that developer can provide custom cookie options. Which is correct?